air-quality-ui/system_tests/ui/summary_page.spec.ts

```diff
@@ -1,78 +1,570 @@
+              pm2_5: { mean: { aqi_level: 6, value: 80 } },
+              pm10: { mean: { aqi_level: 6, value: 160 } },
+              so2: { mean: { aqi_level: 6, value: 800 } },
+            }),
+
+            createMeasurementSummaryAPIResponseData({
+              measurement_base_time: '2024-07-08T012:00:00Z',
+              overall_aqi_level: { mean: 4 },
+              no2: { mean: { aqi_level: 4, value: 125 } },
+              o3: { mean: { aqi_level: 4, value: 135 } },
+              pm2_5: { mean: { aqi_level: 4, value: 26 } },
+              pm10: { mean: { aqi_level: 4, value: 99 } },
+              so2: { mean: { aqi_level: 4, value: 205 } },
+            }),
+          ]
+
+          await summaryPage.setupPageWithMockData(
+            forecastLondonValidTimeArray,
+            measurementsLondonArray,
+          )
+
+          const expectedTableContents: string[][] = [
+            [
+              '3',
+              '6',
+              '-3',
+              '22.5',
+              '80',
+              '08 Jul 09:00',
+              '45.5',
+              '160',
+              '08 Jul 09:00',
+              '100.5',
+              '900',
+              '08 Jul 09:00',
+              '100.5',
+              '400',
+              '08 Jul 09:00',
+              '300.5',
+              '800',
+              '08 Jul 09:00',
+            ],
+          ]
+
+          await summaryPage.assertGridValues(expectedTableContents)
+        })
+
+        test('Verify table shows pollutant data for the timestamp that has the largest deviation - diff 0 - forecast AQI 3, measurement AQI 3', async ({
+          summaryPage,
+        }) => {
+          const forecastLondonValidTimeArray: object[] = [
+            createForecastAPIResponseData(),
+            createForecastAPIResponseData({
+              valid_time: '2024-07-08T03:00:00Z',
+            }),
+            createForecastAPIResponseData({
+              valid_time: '2024-07-08T12:00:00Z',
+            }),
+          ]
+
+          const measurementsLondonArray: object[] = [
+            createMeasurementSummaryAPIResponseData(),
+            createMeasurementSummaryAPIResponseData({
+              measurement_base_time: '2024-07-08T03:00:00Z',
+            }),
+            createMeasurementSummaryAPIResponseData({
+              measurement_base_time: '2024-07-08T12:00:00Z',
+              overall_aqi_level: { mean: 3 },
+              no2: { mean: { aqi_level: 3, value: 119 } },
+              o3: { mean: { aqi_level: 3, value: 129 } },
+              pm2_5: { mean: { aqi_level: 3, value: 24 } },
+              pm10: { mean: { aqi_level: 3, value: 49 } },
+              so2: { mean: { aqi_level: 3, value: 349 } },
+            }),
+          ]
+
+          await summaryPage.setupPageWithMockData(
+            forecastLondonValidTimeArray,
+            measurementsLondonArray,
+          )
+
+          const expectedTableContents: string[][] = [
+            [
+              '3',
```

Comment:
I appreciate the linter or compiler might not let us, but could we structure this array to make it clearer what we're checking in these tests, something like:

[
    '3', '3', '0',                              // Overall API
    '22.5', '24, '08 Jul 12:00',       // PM 2.5 values
    ...etc
]

I don't know if this will work though.